src/components/NameInput.js

```diff
@@ -21,12 +21,13 @@ class NameInput extends React.Component {
 			});
 
 		if (this.props.errors.error) {
+			console.log(this.props);
 			Swal.fire({
 				title: `Error: ${this.props.errors.error.error_code}`,
 				text: `${this.props.errors.error.error} `,
-				icon: 'error',
-				confirmButtonText: 'Continue',
-			});
+				icon: "error",
+				confirmButtonText: "Continue"
+			}).then(this.props.clearErrors());

```

Comment:
You should be able to just `await` `Swal.fire` and call `clearErrors` normally after.